girder_worker/__init__.py

```diff
@@ -209,6 +219,8 @@ def run(task, inputs=None, outputs=None, auto_convert=True, validate=True,
         was provided. Instead, those outputs will be saved to that URI and
         the output binding will contain the location in the ``'uri'`` field.
     """
+    load_plugins()
+
```

Comment:
I feel like `load_plugins()` should go anywhere that might need plugins loaded and not just the `run` function. Other possible entry points are `isvalid`, `load`, `convert`, `format.has_converter`, `format.get_validator_analysis`, and `format.converter_path`. See my related comment on tests.